src/components/article/AllArticles.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { getAllArticles } from '../../actions/articles';
+import './AllArticles.scss';
+
+class AllArticles extends Component {
+  state = {
+    done: false,
+    defaultImg: 'https://tmsnrt.rs/2FNDBT2',
+    defaultText: 'Just a placeholder',
+  };
+
+  async componentDidMount() {
+    const { getAllArticles: getAll } = this.props;
+    await getAll();
+    this.setState({ done: true });
+  }
+
+  handleClick = async () => {
+    const { getAllArticles: getAll } = this.props;
+    await getAll();
+    this.setState({ done: true });
+  };
+
+  renderArticles() {
+    const { allArticles } = this.props;
+    const { done, defaultImg, defaultText } = this.state;
+    if (done) {
+      return allArticles.allArticles.map(article => (
+        <div className="col s12 m6 l6" key={article.id}>
+          <div className="card">
+            <div className="card-image">
+              <img
+                src={article.imgUrl ? article.imgUrl : defaultImg}
+                alt={article.title ? article.title : defaultText}
+              />
+            </div>
+
+            <div className="card-content">
+              <span className="card-title">{article.title}</span>
```

Comment:
Quick observation:

while perusing your awesome work on this feature and testing out the app, I noticed that one can not click on an article card (or even the title) to read the article.

Here would be a good spot to `Link to` to the article itself don't you think?

Reply:
Nice observation. It has been fixed